src/snowflake/cli/api/cli_global_context.py

```diff
@@ -399,19 +188,47 @@ def _should_force_mute_intermediate_output(self) -> bool:
         return self._manager.output_format == OutputFormat.JSON
 
 
-_CLI_CONTEXT_MANAGER: _CliGlobalContextManager | None = None
-_CLI_CONTEXT: _CliGlobalContextAccess | None = None
+_INITIAL_CLI_CONTEXT = _CliGlobalContextManager()
+
+_CLI_CONTEXT_MANAGER: ContextVar[_CliGlobalContextManager] = ContextVar(
+    "cli_context", default=_INITIAL_CLI_CONTEXT
```

Comment:
this was only extracted for debugging, we should hide it again
```suggestion
_CLI_CONTEXT_MANAGER: ContextVar[_CliGlobalContextManager] = ContextVar(
    "cli_context", default=_CliGlobalContextManager()
```